src/main/java/seedu/address/logic/commands/queue/BreakCommand.java

```diff
@@ -17,7 +17,7 @@
     public static final String COMMAND_WORD = "break";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Doctor goes for a break\n"
+            + ": Doctor goes for a break. Avoids directing patients to the doctor based on the room number.\n"
```

Comment:
Please rephrase this. There should be no reference to any sort of room. It only adds a list of staff doctors on duty.

Reply:
Rephrased